packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -11,33 +11,43 @@ import { IconButton, Toolbar } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { addQueryArgs } from '@wordpress/url';
 
-function FullscreenModeClose( { isActive, postType } ) {
-	if ( ! isActive || ! postType ) {
+function FullscreenModeClose( { isActive, postType, isAlwaysShown, isLabeled } ) {
+	if ( ! ( isActive || isAlwaysShown ) || ! postType ) {
 		return null;
 	}
 
+	const label = get(
+		postType,
+		[ 'labels', 'view_items' ],
+		__( 'Back' )
+	);
+
+	const className = isAlwaysShown ? 'edit-post-always-show-close__toolbar' : 'edit-post-fullscreen-mode-close__toolbar';
+
 	return (
-		<Toolbar className="edit-post-fullscreen-mode-close__toolbar">
+		<Toolbar className={ className }>
 			<IconButton
 				icon="arrow-left-alt2"
 				href={ addQueryArgs( 'edit.php', { post_type: postType.slug } ) }
-				label={ get(
-					postType,
-					[ 'labels', 'view_items' ],
-					__( 'Back' )
-				) }
-			/>
+				label={ label }
+			>
+				{ isLabeled ? label : null }
+			</IconButton>
 		</Toolbar>
 	);
 }
 
 export default withSelect( ( select ) => {
 	const { getCurrentPostType } = select( 'core/editor' );
-	const { isFeatureActive } = select( 'core/edit-post' );
+	const { isFeatureActive, getPreference } = select( 'core/edit-post' );
 	const { getPostType } = select( 'core' );
+	const isAlwaysShown = getPreference( 'features' ).alwaysShowCloseButton;
```

Comment:
I think `isFeatureActive` could be used here. I.e. `const isAlwaysShown = isFeatureActive( 'alwaysShowCloseButton' );`

Reply:
Oh Yeah!  That does work, thanks for pointing it out.  I actually tried that first and it did not work for a totally different reason.  I updated it to use `isFeatureActive`.

Reply:
great!